pom.xml

```diff
@@ -6,6 +6,7 @@
         <groupId>org.jboss.tools</groupId>
         <artifactId>parent</artifactId>
         <version>4.4.1.Alpha1-SNAPSHOT</version>
+        <relativePath/>
```

Comment:
on my headless vm it doesn't work due existing repositories definition for p2 layout and the property for them was not resolved. see my previous gist https://gist.github.com/mareknovotny/51eb071d67486671e30df4a479df6e16, where it is combination of property in repository and property in jbosstools-parent. I guess it could be a bug, but obviously suggesting maven the looking into remoting repository (and this takes repository/ies from settings.xml in $USERHOME/.m2/ as first to look instead of repository defined in windup eclipse plugin pom.xml)